xml/OptionPanel.xml

```diff
@@ -1,108 +1,182 @@
+              </VerticalLayout>
+            </Cell>
+            <Cell class="option-button">
+              <Toggle id="showCleanUpHelper" onValueChanged="onClick_toggleOption(showCleanUpHelper)"/>
+            </Cell>
+          </Row>
+
+          <!-- Option: show hand helper -->
+          <Row class="option-text">
+            <Cell class="option-text">
+              <VerticalLayout class="text-column">
+                <Text class="option-header">Show "Hand Helper"</Text>
+                <Text class="description">Never count your hand cards again! This tool does that for you and can even take "Dream-Enhancing Serum" into account. Also includes a button for randomly discard a card.</Text>
+              </VerticalLayout>
+            </Cell>
+            <Cell class="option-button">
+              <Toggle id="showHandHelper" onValueChanged="onClick_toggleOption(showHandHelper)"/>
+            </Cell>
+          </Row>
+
+          <!-- Option: show chaos bag manager -->
+          <Row class="option-text">
+            <Cell class="option-text">
+              <VerticalLayout class="text-column">
+                <Text class="option-header">Show "Chaos Bag Manager"</Text>
+                <Text class="description">Panel for easy addition or removal of chaos tokens to the bag - very useful for EotE because of Frost tokens!</Text>
+              </VerticalLayout>
+            </Cell>
+            <Cell class="option-button">
+              <Toggle id="showChaosBagManager" onValueChanged="onClick_toggleOption(showChaosBagManager)"/>
+            </Cell>
+          </Row>
+
+          <!-- Option: show navigation overlay -->
+          <Row class="option-text">
+            <Cell class="option-text">
+              <VerticalLayout class="text-column">
+                <Text class="option-header">Show "Navigation Overlay"</Text>
+                <Text class="description">This enables an overlay for quickly navigating to various points on the table.</Text>
+              </VerticalLayout>
+            </Cell>
+            <Cell class="option-button">
+              <Toggle id="showNavigationOverlay" onValueChanged="onClick_toggleOption(showNavigationOverlay)"/>
+            </Cell>
+          </Row>
+        </TableLayout>
+      </VerticalScrollView>
+    </Cell>
+  </Row>
 
-      <!-- Option: show chaos bag manager -->
-      <HorizontalLayout class="group-content">
-        <VerticalLayout class="text-column">
-          <Text class="option-header">Show "Chaos Bag Manager"</Text>
-          <Text class="description">Panel for easy addition or removal of chaos tokens to the bag - very useful for EotE because of Frost tokens!</Text>
-        </VerticalLayout>
-        <Toggle id="showChaosBagManager" onValueChanged="onClick_toggleOption(showChaosBagManager)"/>
+  <!-- Buttons: Defaults and Close -->
+  <Row preferredHeight="50">
+    <Cell>
+      <HorizontalLayout minHeight="50" flexibleHeight="0" padding="5" spacing="5">
```

Comment:
Minor, but I think the buttons don't need to be this large.  Lots of dead space on them... Maybe shrink them and leave empty space in between?

Reply:
Agreed, gonna change that in the next update